src/SDKs/ContainerRegistry/ContainerRegistry.Tests/Helpers/ContainerRegistryTestUtilities.cs

```diff
@@ -1,17 +1,22 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-using System;
-using System.Linq;
-using System.Collections.Generic;
+using Microsoft.Azure.Management.ContainerRegistry;
```

Comment:
@djyou sort and remove unused usings

Reply:
This is already sorted and cleaned as per VS2017, or is there is different schema SDK uses?